tests/unit_tests/test_analytics/test_sentry.py

```diff
@@ -0,0 +1,313 @@
+import os
+import time
+from unittest import mock
+
+import pytest
+import wandb
+import wandb.analytics
+import wandb.env
+from sentry_relay import MetricRelayServer
+from sentry_sdk.utils import sentry_sdk
+
+EXTERNAL_SENTRY_DSN_KEY = "EXTERNALKEY"
+EXTERNAL_SENTRY_PROJECT = "123456"
+INTERNAL_SENTRY_DSN_KEY = "INTERNALKEY"
+INTERNAL_SENTRY_PROJECT = "654321"
+API_SERVER_ADDRESS = "127.0.0.1:{port}"
+SENTRY_DSN_FORMAT = "http://{key}@{address}/{project}"
+WANDB_DSN_URL_FORMAT = SENTRY_DSN_FORMAT.format(
+    key=INTERNAL_SENTRY_DSN_KEY,
+    address=API_SERVER_ADDRESS,
+    project=INTERNAL_SENTRY_PROJECT,
+)
+EXTERNAL_DSN_URL_FORMAT = SENTRY_DSN_FORMAT.format(
+    key=EXTERNAL_SENTRY_DSN_KEY,
+    address=API_SERVER_ADDRESS,
+    project=EXTERNAL_SENTRY_PROJECT,
+)
```

Comment:
now, that i look at this what do you think about only keeping:
```
SENTRY_DSN_FORMAT = "http://{key}@{address}/{project}"
```
i know there is some repetition, but i think this is the only interesting part of all these vars

Reply:
personally I think it is best to keep these defined as constants in the file. My rationale is

- If in the future more tests are added, reuse of these are easier, and avoids possibly mistyping.
- If for any reason we need to change one of the values, it can be changed from a single place

I think if we do want to remove the clutter from this file, we can move this to something like `constants.py` file or similar to avoid having too many things in this test file. Thoughts?

Reply:
actually the opposite i would rather it be closer to the test and if there is a typo shouldn't it be caught in the test directly? these are all just values you use for the test, no special meaning as far as i can see